CONTRIBUTING.md

```diff
@@ -1,29 +1,81 @@
 
-All of these are automatically built and deployed by our [Jenkins build servers](https://builds.renegadetech.mixxx.org/) whenever a change is committed. You are welcome to open a pull request in any of these repositories.
+We use [Zulip chat](https://mixxx.zulipchat.com/) to discuss and coordinate everything related to Mixxx development (and more!).
 
-## Git Workflow ##
-### All Contributors ###
-* Each feature/bug fix should be done on its own Git branch so they can be reviewed and merged independently. Refer to [Using Git](https://mixxx.org/wiki/doku.php/using_git) for how to do this. Please ask for help on [Zulip](https://mixxx.zulipchat.com/) if you have questions about using Git after reading that page.
-* Commits should be as small as they can while still building. The smaller the commit, the easier it is to review. It also makes it easier to revert if it is later identified as the source of a bug. If you have lots of changes that you need to commit, a [GUI Git client](https://git-scm.com/downloads/guis) can be helpful for picking out specific changes for multiple small commits.
+- We encourage you to say hi on the [introduce yourself stream](https://mixxx.zulipchat.com/#narrow/stream/109123-introduce-yourself) and share what you would like to contribute.
+- If you need help with your contribution, the Mixxx codebase or anything else related to Mixxx development don't hesitate to ask on the [development help stream](https://mixxx.zulipchat.com/#narrow/stream/247620-development-help).
+- The [development stream](https://mixxx.zulipchat.com/#narrow/stream/109171-development) is where day to day discussion about code, new features and releases happens.
+
+We also have lots of additional information for users and developers on the [Mixxx wiki](https://github.com/mixxxdj/mixxx/wiki), make sure to check it out if there's some piece of information you can't find.
+
+## How can I contribute? ##
+
+There are lots of ways you can contribute to Mixxx. Even if you don't know how to program, there's plenty of things you can do to help:
+
+ - [Report a bug](https://github.com/mixxxdj/mixxx/wiki/Reporting%20bugs)
+ - [Test the latest developments](https://github.com/mixxxdj/mixxx/wiki/Testing)
+ - [Translate Mixxx to other languages](https://github.com/mixxxdj/mixxx/wiki/Internationalization)
+ - [Improve the Mixxx manual](https://github.com/mixxxdj/manual#readme)
+ - [Improve the Mixxx website](https://github.com/mixxxdj/website)
+
+If you have some coding skills, or you are willing to learn them you can do even more:
+
+ - [Add a mapping for a new controller](https://github.com/mixxxdj/mixxx/wiki/Contributing%20mappings)
+ - [Fix bugs or program new features](#Your-first-Mixxx-code-contribution)
+
+We welcome programmers of all backgrounds and skill levels, and our community will do their best to help you with any problems you encounter.
+
+## Your first Mixxx code contribution ##
+
+### Compile Mixxx for the first time ###
```

Comment:
We need to make sure that contributors create a GitHub account + a Mixxx fork first. Than clone the private repro.
Following the links here doe not emphasis this.
It is the most common issue in contributions from not experienced GitHub users.

Reply:
I updated the linked wiki pages explaining this. 

We can go further and just instruct people to fork, not to clone our repo. What do you think?

Reply:
Yes, good idea

Reply:
Done